infrastructure/sentry-subgraph/schema/events/PoolFactory.graphql

```diff
@@ -17,6 +17,7 @@ type PoolInfo @entity(immutable: false) {
   ownerLatestUnstakeRequestCompletionTime: BigInt!
   createdTimestamp: BigInt!
   poolChallenges: [PoolChallenge!]! @derivedFrom(field: "pool")
+  submissions: [PoolSubmission!]! @derivedFrom(field: "poolInfo")
 }
```

Comment:
Why is poolChallenges derived from field "pool", but poolSubmissions are derived from "poolInfo"?

Reply:
I did not want to change anything from the already reviewed field names as I did not know about possible dependencies.

I only added the mapping from the PoolInfo side so we can query for PoolInfo and attach a list of submissions.